src/commands/junit/api.ts

```diff
@@ -32,10 +31,17 @@ export const uploadJUnitXML = (request: (args: AxiosRequestConfig) => AxiosPromi
     fileName = 'default_file_name'
   }
 
+  const reportTagsAndMetrics: Record<string, any> = {
+    tags: jUnitXML.reportTags,
+    metrics: jUnitXML.reportMetrics,
+  }
+
   const metadata: Record<string, any> = {
```

Comment:
maybe rename this now that it has an internal attribute 'metadata' it might be confusing